packages/blaze/view.js

```diff
@@ -509,21 +512,6 @@ Blaze._destroyNode = function (node) {
     Blaze._DOMBackend.Teardown.tearDownElement(node);
 };
 
-// Are the HTMLjs entities `a` and `b` the same?  We could be
-// more elaborate here but the point is to catch the most basic
-// cases.
-Blaze._isContentEqual = function (a, b) {
```

Comment:
I could have simply renamed the new function as ._isContentEqual, except this function is meant for DOM comparison, and we are comparing objects / primitives, so to avoid confusion I broke with the past.